client/account-recovery/lost-password/lost-password-form/style.scss

```diff
@@ -0,0 +1,22 @@
+.lost-password-form__title {
+	font-size: 1.5em;
+	font-weight: bold;
```

Comment:
For our system font usages (sans-serif) we need to specify `weight: 600` since we use semibold and not bold.